week0004/hoangdangduy/java/SortUtil.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+
+public class SortUtil {
+
+	public ArrayList<Client> sort(ArrayList<Client> listClient) {
+		Collections.sort(listClient, new Comparator<Client>() {
+
+			@Override
+			public int compare(Client client1, Client client2) {
+				return client1.getFname().compareTo(client2.getFname());
```

Comment:
Yes. Do my code process wrong?. First, I sort client order by fname then I sort lname when have similar fname and email do same before.